src/components/toolbar/search-component.ts

```diff
@@ -81,20 +118,30 @@ export function view(
         ? html.div(
             {
               class: 'result-container',
+              scrollTop: (_state.focusedEl && _state.keyPress) ?
```

Comment:
Good work figuring this out too. The `60` is the height of the result element? Can you make it a constant so it's clear what the 60 is exactly.